pkgs/development/python-modules/behave/default.nix

```diff
@@ -17,6 +17,10 @@ buildPythonApplication rec {
   };
 
   checkInputs = [ pytestCheckHook mock pathpy pyhamcrest pytest-html ];
+  doCheck = false;
```

Comment:
please also include a comment as to why tests need to be disabled

Reply:
Done